src/tests/suite/daos_epoch_io.c

```diff
@@ -892,22 +925,23 @@ cmd_parse_update_fetch(test_arg_t *arg, int argc, char **argv, int opc,
 	struct test_update_fetch_arg	*uf_arg;
 	daos_recx_t			*recxs = NULL;
 	int				*values = NULL;
-	unsigned int			 recx_num = 0;
+	unsigned int			recx_num = 0;
 	char				*dkey = NULL;
 	char				*akey = NULL;
-	daos_epoch_t			 epoch = 1;
-	bool				 array = true;
-	int				 opt;
-	int				 rc = 0;
+	int				tx = 1;
+	bool				array = true;
+	int				opt;
+	int				rc = 0;
 
 	static struct epoch_io_cmd_option options[] = {
 		{"--dkey",	true,	'd'},
 		{"--akey",	true,	'a'},
 		{"--single",	false,	's'},
-		{"--epoch",	true,	'e'},
+		{"--tx",	true,	'e'},
```

Comment:
not sure why you rename epoch to tx, or maybe "hlc"/"tx_hlc" or sth else is better name
and you are renaming epoch rather than removing it as said in commit msg.